packages/main/src/themes/ToolbarPopover.css

```diff
@@ -0,0 +1,34 @@
+.ui5-overflow-list {
+	display: flex;
+	flex-direction: column;
+	justify-content: center;
+	align-items: center;
+	padding: 0 0.5rem;
+}
+
+.ui5-overflow-list--alignleft .ui5-tb-popover-item {
+	justify-content: flex-start;
+}
+
+.ui5-overflow-list--alignleft .ui5-tb-popover-item[icon-only]{
+	margin-left: 0.625rem;
```

Comment:
margin-inline-start. 
CSS logical props are recommended, otherwise you need to write RTL styles